lib/capsize/deployer.rb

```diff
@@ -123,6 +123,28 @@ def self.cvs_root_defintion?(val)
       val.index(':') == 0 && val.scan(":").size > 1
     end
 
+    def print_parameter(parameter)
+      name = parameter.name
+      val = parameter.value
+
+      val = deployment.prompt_config[name] if parameter.prompt?
+
+      return nil if val.nil?
+      val.strip!
+      return set_string_parameter(name, val) if val =~ /\A\d+\D+/
+
+      case val[0]
+      when ":", "%", "[", "{", /\d/
+        "set :#{name}, #{val}"
+      else
+        set_string_parameter(name, val)
+      end
+    end
+
+    def set_string_parameter(name, val)
+      "set :#{name}, '#{val}'"
```

Comment:
I used it twice, once under `else` and once in the guard clause.  Though I have thought of something I might have missed in this method so I will probably be making a method for 138 as well.